README.md

```diff
@@ -211,6 +211,7 @@ similar to `ip` (inner paragraph) and `ap` (a paragraph).
 lua <<EOF
 require'nvim-treesitter.configs'.setup {
   textobjects = {
+    enable = true,
```

Comment:
This isn't needed. They stopped working because the module was moved to a submodule. See the examples in the docs for the new settings.